frontend/src/actions/story.js

```diff
@@ -27,7 +29,10 @@ export const submitStory = (dto) => async (dispatch) => {
     },
   });
 
-  if (!response.error) history.push(nextPage);
+  if (!response.error) {
+    if (travels.length) dispatch(submitTravels(travels, response.id, nextPage));
```

Comment:
Not for making the change now, but seeing this makes me wonder: Is it a good pattern to do the navigation inside the action? Without knowing a lot of js/react world, it seems to me that no, but 🤷

Reply:
The action ends up needing to know more things (in this case the `nextPage`) and it seems a little off of its intention/responsibility, but again: 🤷

Reply:
It's one of those cases of "it's not a good practice for redux but it is needed by React" - and that's why other frameworks are gaining ground on React's old skeleton!